server/lib/PasswordUtils.hh

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace server {
+
+class PasswordUtils
+{
+  public:
+    static std::string
+    hashPassword(std::string const& password);
+
+    static bool
+    checkHash(std::string const& hash, std::string const& password);
+};
```

Comment:
Is there any need for a class here? Those functions would be better in a namespace.